src/v2/ion/i18nTransformer.js

```diff
@@ -122,6 +122,35 @@ const I18N_OVERRIDE_MAPPINGS = {
   'oie.session.expired' : 'oie.idx.session.expired',
 };
 
+const getDisplayName = (remediation, defaultName) => {
+  return remediation.relatesTo?.value?.displayName || defaultName;
+};
+
+const I18N_PARAMS_MAPPING = {
+  [FORMS.ENROLL_AUTHENTICATOR]: {
+    [AUTHENTICATOR_KEY.ON_PREM]: {
+      getParam: getDisplayName,
+      defaultParam: loc('oie.on_prem.authenticator.default.vendorName', 'login'),
+    },
+    [AUTHENTICATOR_KEY.RSA]: {
+      getParam: getDisplayName,
+      defaultParam: loc('oie.on_prem.authenticator.default.vendorName', 'login'),
```

Comment:
nit: should this point to `oie.rsa.authenticator.default.vendorName` ?

Reply:
good catch! I use the same default vendor name for both since they are practically the same.. but `factor.totpHard.rsaSecurId` makes more sense here. Thank you!